packages/forms/package.json

```diff
@@ -10,7 +10,7 @@
     "test:watch": "talend-scripts test --watch",
     "test:cov": "talend-scripts test --coverage",
     "test:demo": "build-storybook",
-    "lint:es": "talend-scripts lint:es",
+    "lint:es": "talend-scripts lint:es --ignore-pattern deprecated",
```

Comment:
Let's ignore the deprecated Form V1 to avoid tons of linter errors that we won't fix.